Apps/W1/External File Storage - SharePoint Connector/app/src/ExtSharePointAccount.Table.al

```diff
@@ -71,25 +87,131 @@ table 4580 "Ext. SharePoint Account"
     var
         UnableToGetClientMsg: Label 'Unable to get SharePoint Account Client Secret.';
         UnableToSetClientSecretMsg: Label 'Unable to set SharePoint Client Secret.';
+        UnableToGetCertificateMsg: Label 'Unable to get SharePoint Account Certificate.';
+        UnableToSetCertificateMsg: Label 'Unable to set SharePoint Certificate.';
+        UnableToGetCertificatePasswordMsg: Label 'Unable to get SharePoint Account Certificate Password.';
+        UnableToSetCertificatePasswordMsg: Label 'Unable to set SharePoint Certificate Password.';
 
     trigger OnDelete()
     begin
-        if not IsNullGuid(Rec."Client Secret Key") then
-            if IsolatedStorage.Delete(Rec."Client Secret Key") then;
+        TryDeleteIsolatedStorageValue(Rec."Client Secret Key");
+        TryDeleteIsolatedStorageValue(Rec."Certificate Key");
+        TryDeleteIsolatedStorageValue(Rec."Certificate Password Key");
     end;
 
+#if CLEAN28
```

Comment:
26.0 is on the road.
The normal timeline would be BC27.0 as this the current branch and `CLEAN27`.

Reply:
@JesperSchulz is there any way to set this to internal faster than the normal cycle? 
tinestaric has right, this is a security issue. Every App kan use the credentials of the connector.

Reply:
Absolutely. We can make breaking changes in such occasions. Depending on the severity, we can hotfix too. Please reach out to me on teams for a discussion.